### DISCUSSION TITLE
Face Landmark

### DISCUSSION THREAD

**Discussion by guzuligo** [Mikubill/sd-webui-controlnet#630] [Q&A]
There is a new Face Landmark detection for controlNet:
https://huggingface.co/georgefen/Face-Landmark-ControlNet

Are there plans to implement this?

**Comment by fofr** [comment]
You can put the model into your models directory and it seems to work, but I don't know how to get the preprocessor working.

**Reply by guzuligo** [reply]
preprocessor actually needs to be programmed for it to work. The code there isn't ready to be copy pasted into the annotators directory, unfortunately.

**Reply by 3dcinetv** [reply]
subscribing to this thread. I'm interested in installing this on my SD web Ui. Thanks

**Comment by OedoSoldier** [comment]
#637
